src/stdlib/stdlib/std/internal/math.tact

```diff
@@ -124,16 +124,16 @@ inline fun random(min: Int, max: Int): Int {
 
 /// Global function.
 ///
-/// Computes and returns the minimum of two `Int` values `x` and `y`.
+/// Computes the minimum of two `Int` values `x` and `y`.
 ///
 /// ```tact
 /// fun examples() {
 ///     min(1, 2);        // 1
 ///     min(2, 2);        // 2
 ///     min(007, 3);      // 3
-///     min(0x45, 3_0_0); // 69, nice
+///     min(0x42, 3_0_0); // 66, nice
```

Comment:
I'd remove the "nice" here because it's no longer a meme due to the number change